app/controllers/versions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class VersionsController < ApplicationController
+  def index
+    @versions = PaperTrail::Version.where(item_id: params.require(:item_id), item_type: params.require(:item_type))
```

Comment:
if nothing is stored that matches this type then nothing will be found ...
we might have to later lock this down if we only want certain types ...

On Tue, Apr 26, 2016 at 4:40 PM, Roman Sandler <notifications@github.com>
wrote:

> In app/controllers/versions_controller.rb
> <https://github.com/zendesk/samson/pull/876#discussion_r61183102>:
>
> > @@ -0,0 +1,5 @@
> > +class VersionsController < ApplicationController
> > +  def index
> > +    @versions = PaperTrail::Version.where(item_id: params.require(:item_id), item_type: params.require(:item_type))
>
> thinking more about this, how confident are we that PaperTrail won't ever
> const_get(item_type)? Do we need a whitelist?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/876/files/69572b5fe7d2940de81d948e4034a3a797d5890c#r61183102>
>